src/status_im2/contexts/quo_preview/wallet/account_card.cljs

```diff
@@ -0,0 +1,140 @@
+    :customization-color :turquoise
+    :type                :default
+    :emoji               "🎲"}
+   {:id               2
+    :name             "Ben’s fortune"
+    :balance          "€2,269.12"
+    :percentage-value "16.9%"
+    :amount           "€570.24"
+    :watch-only?      true
+    :type             :watch-only
+    :emoji            "💸"}
+   {:id                  3
+    :name                "Alisher account"
+    :balance             "€2,269.12"
+    :percentage-value    "16.9%"
+    :amount              "€570.24"
+    :customization-color :purple
+    :type                :default
+    :emoji               "💎"}
+   {:id                  4
+    :type                :add-account
+    :customization-color :blue
+    :handler             #(js/alert "Add account pressed")}])
+
+(def descriptor
+  [{:label   "Type:"
+    :key     :type
+    :type    :select
+    :options [{:key   :default
+               :value "Default"}
+              {:key   :watch-only
+               :value "Watch Only"}
+              {:key   :add-account
+               :value "Add Account"}]}
+   {:label "Show FlatList:"
+    :key   :show-flatlist
+    :type  :boolean}
+   {:label   "Customization color:"
+    :key     :customization-color
+    :type    :select
+    :options (map (fn [[color-kw _]]
+                    {:key   color-kw
+                     :value (name color-kw)})
+                  colors/customization)}
+   {:label "Name:"
+    :key   :name
+    :type  :text}
+   {:label "Balance:"
+    :key   :balance
+    :type  :text}
+   {:label "Emoji:"
+    :key   :emoji
+    :type  :text}])
+
+(defn- separator
+  []
+  [rn/view {:style {:width 12}}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:name                "Alisher account"
+                             :balance             "€2,269.12"
+                             :percentage-value    "16.9%"
+                             :amount              "€570.24"
+                             :customization-color :blue
+                             :type                :default
+                             :emoji               "💎"})]
+    (fn []
+      [rn/view
+       {:style {:flex 1}}
+       [rn/view
+        {:style {:margin-vertical 40
+                 :padding-left    40
+                 :flex-direction  :row
+                 :align-items     :center}}
+        [text/text
+         {:size   :heading-1
+          :weight :semi-bold} "Account card"]
+        [rn/view
+         {:style {:width            20
+                  :height           20
+                  :border-radius    60
+                  :background-color colors/success-50
+                  :align-items      :center
+                  :justify-content  :center
+                  :margin-left      8}}
+         [icon/icon :i/check {:color colors/white :size 16}]]]
+       [rn/view {:style {:flex 1}}
+        [preview/customizer state descriptor]]
+       (if (:show-flatlist @state)
```

Comment:
We should strictly follow the Design System. The Account Card component is a single component with many variations. The fact that it can be displayed in a horizontally scrollable list is not specified in the Figma component, this is actually a feature of the real wallet screen using the component.

This is also important to not confuse designers reviewing the preview area.

Therefore, I think we should remove the flatlist option.

Reply:
I understand your point, but that was required before. I will talk to the team and come up with a decision

Reply:
Hmm, there is a bit of confusion here. @ilmotta is right for getting this pr merged.
However @FFFra was asked to add it as part of the interview task - but since the interview task is complete let's go with what we want in the codebase, i.e what @ilmotta is suggesting 👍

Reply:
To be removed =)

Reply:
It's removed. I also updated the videos for this PR =)

Reply:
I see guys, thanks. So next time it would help if we have the requirements up-to-date in the PR description, in case they override the guidelines/conventions.